menas/src/test/scala/za/co/absa/enceladus/menas/integration/controllers/RunApiIntegrationSuite.scala

```diff
@@ -28,6 +29,7 @@ import za.co.absa.enceladus.model.{Run, SplineReference}
 
 @RunWith(classOf[SpringRunner])
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles(Array("withEmbeddedMongo"))
```

Comment:
The profile decoration could have probably been used in the base class instead - with the same effect. This makes it more verbose.